bindings/python/iota_sdk/types/burn.py

```diff
@@ -23,33 +24,34 @@ class Burn:
         The native tokens to burn
     """
 
-    aliases: Optional[List[str]]
-    nfts: Optional[List[str]]
-    foundries: Optional[List[str]]
+    aliases: Optional[List[HexStr]]
+    nfts: Optional[List[HexStr]]
+    foundries: Optional[List[HexStr]]
     nativeTokens: Optional[List[NativeToken]]
 
-    def add_alias(self, alias: str) -> Burn:
+    def add_alias(self, alias: HexStr) -> Burn:
         if self.aliases is None:
             self.aliases = []
         self.aliases.append(alias)
         return self
 
-    def add_nft(self, nft: str) -> Burn:
+    def add_nft(self, nft: HexStr) -> Burn:
         if self.nfts is None:
             self.nfts = []
         self.nfts.append(nft)
         return self
 
-    def add_foundry(self, foundry: str) -> Burn:
+    def add_foundry(self, foundry: HexStr) -> Burn:
         if self.foundries is None:
             self.foundries = []
         self.foundries.append(foundry)
         return self
 
     def add_native_token(self, native_token: NativeToken) -> Burn:
```

Comment:
Is it a python thing that params are snake_case and fields camelCase?

Reply:
Probably not, I think @qrayven did it like this because in the serialized version it needs to be camelCase

Reply:
Ah, right